src/main/java/nl/novi/techiteasy1121/controllers/TelevisionController.java

```diff
@@ -1,40 +1,43 @@
 package nl.novi.techiteasy1121.controllers;
 
```

Comment:
Zorg ervoor dat je JavaDoc toevoegt aan je methoden en klassen. Dit helpt andere ontwikkelaars (en jezelf in de toekomst) om de functionaliteit snel te begrijpen.